supersim.go

```diff
@@ -66,9 +66,9 @@ func NewSupersim(log log.Logger, envPrefix string, closeApp context.CancelCauseF
 		return nil, fmt.Errorf("failed to create orchestrator")
 	}
 
-	adminServer := admin.NewAdminServer(log, cliConfig.AdminPort, &networkConfig)
+	a := admin.NewAdminServer(log, cliConfig.AdminPort, &networkConfig, o.L2ToL2MessageIndexer())
```

Comment:
since now there's a dependency on the orchestrator's internal properties, we should move the admin server start / end logc into orchestrator